gtfs/spec/en/reference.md

```diff
@@ -380,6 +382,30 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### occupancies.txt
+
+File: **Optional**
+
+Usual or expected in-vehicle occupancy levels.
+
+For describing historical average occupancies at least 4 weeks of occupancy data should be averaged across each unique day of the week, valid for up to 2 weeks thereafter in `occupancies.txt`.
+
+Methods for occupancy forecasting used to populated `occupancies.txt` are not strictly specified. Beyond the guidelines given above, it is the responsibility of data producers to ensure that the data provided are meaningful. 
```

Comment:
Rather than phrasing this as a normative statement, could the first sentence be treated as guidance like the second?

There are many providers who have conducted analyses and determined that the most predictive values come from durations shorter than an arbitrary four-week period, or a given number of historical trips.

Reply:
The word "[should](https://datatracker.ietf.org/doc/html/rfc2119#section-3)" denotes the first sentence as a recommendation. Do you have a proposal for a rephrase?

Reply:
That's valid, although I took the "should" to apply to the average and not the amount of data.

My personal preference would be to explicitly state it as a recommendation and swap the two sentences. I think that clarifies the suggestion to producers, while reminding consumers that other mechanisms might be employed in the computation of the data. (e.g. I wouldn't want a consumer to display that as the methodology for how metrics are computed, leaving us to be an exception.)

```suggestion
Methods for occupancy forecasting used to populated `occupancies.txt` are not strictly specified. Beyond the guidelines given below, it is the responsibility of data producers to ensure that the data provided are meaningful.

For describing historical average occupancies, it is recommended that at least 4 weeks of occupancy data be averaged across each unique day of the week, and valid for up to 2 weeks thereafter in `occupancies.txt`.
```

Reply:
I applied your suggestion :) Please sign the [CLA](https://cla.developers.google.com/about/google-individual). If you can't for whatever reason let me know.